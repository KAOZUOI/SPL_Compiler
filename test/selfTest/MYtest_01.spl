int main() {
  int rect;
  rect.length = 10;
  rect.width = 20;
  return 0;
}