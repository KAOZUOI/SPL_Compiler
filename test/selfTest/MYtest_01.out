Error type 13 at Line 3: the type of operands of "." must be a structure.
Error type 13 at Line 4: the type of operands of "." must be a structure.